### PR TITLE
Update client and donor model validations

### PR DESCRIPTION
**Pull request by emarti43**
## What this PR does
- updates validations for Donor and Client models to match frontend validation constraints.

## How to test changes
*  run the rails console (`docker-compose run web rails c`) and check for any anomalies for validation
* ensure that database seeding still works

## Additional Notes
* not sure if `business_license` can be modified to be even stricter? Not sure what is the business license format we're sticking to. I've made some changes to the current seeds in case the requirements are enough

### CODE REVIEW

File: app/models/donor.rb [hunk 1]
```diff
@@ -4,14 +4,18 @@ class Donor < ApplicationRecord
 	has_many :donations
 	has_many :claims, through: :donations
 	accepts_nested_attributes_for :claims
-	
-	validates :organization_name, presence: true
-	validates :email, uniqueness: { case_sensitive: false }
-	validates :business_license, presence: true
+
+	validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }
```
Comment (renburnett) [inline app/models/donor.rb:8]: Awesome validation 👍

File: app/models/donor.rb [hunk 2]
```diff
@@ -4,14 +4,18 @@ class Donor < ApplicationRecord
 	has_many :donations
 	has_many :claims, through: :donations
 	accepts_nested_attributes_for :claims
-	
-	validates :organization_name, presence: true
-	validates :email, uniqueness: { case_sensitive: false }
-	validates :business_license, presence: true
+
+	validates :email, format: { with: URI::MailTo::EMAIL_REGEXP } 
+	validates :first_name, presence: false
```
Comment (renburnett) [inline app/models/donor.rb:9]: Why not `presence: true` ?

Comment (emarti43) [reply app/models/donor.rb]: I had originally aligned the requirements to the design we had prior to the pre-alpha. I just didn't revert that change when pushing the branch 😅. It should be good now!

Comment (renburnett) [reply app/models/donor.rb]: Ah gotcha, no worries at all! 👍

**Comment by Gauri-BGP** [comment]
just making sure, Are these in alignment with pre alpha requirements?

**Comment by emarti43** [comment]
These are not in alignment with pre-alpha requirements. Gonna close this PR since these validations have been added with the most recent PR merge.

**Pull request closed without merging**